doc/packages/emacs.section.md

```diff
@@ -1,8 +1,16 @@
 # Emacs {#sec-emacs}
 
+[Emacs](https://www.gnu.org/software/emacs/) is the advanced, extensible, customizable, self-documenting editor.
+
+At its core is an interpreter for Emacs Lisp (shortly, Elisp), a dialect of the Lisp programming language with extensions to support text editing.
+
 ## Configuring Emacs {#sec-emacs-config}
 
-The Emacs package comes with some extra helpers to make it easier to configure. `emacs.pkgs.withPackages` allows you to manage packages from ELPA. This means that you will not have to install that packages from within Emacs. For instance, if you wanted to use `company` `counsel`, `flycheck`, `ivy`, `magit`, `projectile`, and `use-package` you could use this as a `~/.config/nixpkgs/config.nix` override:
+Nixpkgs provides a framework that leverages Emacs via Nix.
```

Comment:
> The Emacs package comes with some extra helpers to make it easier to configure.

vs

> Nixpkgs provides a framework that leverages Emacs via Nix.

I find the old sentence is more easy to read/understand.

Also, `a framework that leverages Emacs via Nix` feels wrong to me.  Maybe `a framework to leverage Emacs via Nix`?  I am not a native English speaker though.